### ISSUE TITLE
import-file-stix connector Error: unable to import required numpy module.

### ISSUE THREAD

**Issue by symsal** [OpenCTI-Platform/connectors#846]
## Description

Error: unable to import required numpy module.

See https://pypi.python.org/pypi/numpy.Listing Threat-Actors with filters null.

OpenCTI API is not reachable. Waiting for OpenCTI API to start or check your configuration...

Result in looping task

## Environment

1. OS (where OpenCTI server runs): { docker }
2. OpenCTI version: { 5.3.13}
3. OpenCTI client: { frontend }
4. Other environment details:

**Comment by SamuelHassine** [comment]
Hello @symsal,

Can you please share the logs of the connector. Numpy is not an OpenCTI dependency.

Kind regards,
Samuel

**Comment by SamuelHassine** [comment]
Yes, it just seems your connector has no access to your platform.
numpy is not used anywhere in our source code. Are you sure you are using the latest version of the connector?

**Comment by SamuelHassine** [comment]
Hello @symsal,

Just delete the file. It was a bug in previous versions of the platform that looping tasks.

Kind regards,
Samuel

**Comment by ckane** [comment]
FYI, this should be fixed in PR #1022 - the offending module was the `maec` Python module, which started including `numpy` in one part of it
